src/pages/__snapshots__/Bootcamp.spec.tsx.snap

```diff
@@ -13,6 +13,20 @@ exports[`Bootcamp renders as expected 1`] = `
       <div
         className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
```

Comment:
I don't see value with snapshot 🌊

Reply:
why is that?

Reply:
Testing = confidence in app. 
Snapshot doesn't bring any.